common/lib/xmodule/xmodule/modulestore/xml_importer.py

```diff
@@ -107,6 +107,12 @@ def import_static_content(
 
             policy_ele = policy.get(asset_key.path, {})
             displayname = policy_ele.get('displayname', filename)
+
+            # During export display name is used to create files, it should not contain '/'. TNL-2669
+            # strip away slashes from path from the name
+            if '/' in displayname:
+                displayname = displayname.replace('/', '_')
```

Comment:
what if a course has two files: `a/b/c` _and_ `a_b_c`? Will there be two `a_b_c`s now? Or will one clobber the other?